test/apps/uct_info/Makefile.in

```diff
@@ -4,6 +4,8 @@
 # See file LICENSE for terms.
 #
 
+NUMA_LIBS = @NUMA_LIBS@
```

Comment:
why needed?
maybe we can add numa to uct_info after it no longer depends on libnuma.so ?

Reply:
Removed in next PR...

Reply:
question why is it needed in this PR?

Reply:
to handle